packages/webpack/src/runtime/runtime.js

```diff
@@ -261,6 +270,20 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     // wrap webpack runtime for policy check and hardening
     const policyRequire = wrapRequireWithPolicy(__webpack_require__, resourceId)
 
+    /**
+     * Wrapped chunk loader
+     *
+     * @param {string | number} chunkId
+     */
+    policyRequire.e = (chunkId) => {
+      // Chunk resolution at runtime may be a lookup table or a string concatenation. In the latter case it can be fooled into loading any file from under publicPath. That's why we only want to accept known chunks.
+      if (!LAVAMOAT.kch.includes(chunkId)) {
```

Comment:
could this array get large enough to want to convert it to a `Set`?

Reply:
All other stuff that happens with chunks is orders of magnitude more runtime complexity. Each chunk must be added to the page and executed and so on. This is most likely to contain between 0 and 20 items for all reasonable apps